CompositionRoot/CompositionRoot.cs

```diff
@@ -1,20 +1,20 @@
 ﻿using Autofac;
 using Autofac.Core;
 
-namespace Messerli.CompositionRootBuilder
+namespace Messerli.CompositionRoot
 {
-    public sealed class CompositionRootBuilder
+    public sealed class CompositionRoot
```

Comment:
Class should still be called `CompositionRootBuilder`

Reply:
Addressed in: https://github.com/messerli-informatik-ag/composition-root/pull/2/commits/d331ba69cf6b9235fcf22a4405d6ebaebe077d10